.github/workflows/lint-pipelines.yml

```
@@ -0,0 +1,20 @@
+name: Lint Deployment Pipelines
+
+on:
+    pull_request:
+    push:
+        branches: [main, test-me-*]
+
+concurrency:
+    group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+    cancel-in-progress: true
+
+jobs:
+    lint:
+        name: Lint GoCD Pipelines
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c  # v3
```

Since GitHub's `actions` repo is a trusted vendor, we do not pin SHAs here. See the other actions.

```suggestion
            - uses: actions/checkout@v3  # v3
```

For readability, I would also use the version string for our own actions (i.e. `getsentry/*`)